include/platform/platform.h

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This file seems specific to the examples.  Should this be in `examples/include`?